rpcs3/rpcs3qt/gui_settings.h

```diff
@@ -175,6 +175,7 @@ namespace gui
 
 	const gui_save l_tty       = gui_save(logger, "TTY",       true);
 	const gui_save l_level     = gui_save(logger, "level",     static_cast<uint>(logs::level::success));
+	const gui_save l_prefix    = gui_save(logger, "prefix_on", false);
```

Comment:
Why default is false?

Reply:
Because it's better for end users as it means that they get better message stacking by default. Those who find the messages interesting can still manually enable the prefix.